### DISCUSSION TITLE
Issues with AlpineJS @click

### DISCUSSION THREAD

**Discussion by JDavidVR** [breezefront/community#220] [1. Help] [answered]
Hi! I'm adding alpineJS, this is my config:

{Vendor}/AlpineJs/view/frontend/layout/breeze_default.xml

```xml
<body>
        <referenceBlock name="breeze.js">
            <arguments>
                <argument name="bundles" xsi:type="array">

                    <item name="default" xsi:type="array">
                        <item name="items" xsi:type="array">
                            <item name="alpinejs" xsi:type="string">{Vendor}_AlpineJs/lib/alpine</item>
                        </item>
                    </item>

                </argument>
            </arguments>
        </referenceBlock>
    </body>
```

in general it's working well, but when I try to use @click somehow breeze is removing it from .phtml file

here is an alpineJs example:

```html
<div>
    <div x-data="{ open: false }">
        <button @click="open = ! open">Toggle</button>
        <div x-show="open" @click.outside="open = false">Contents...</div>
    </div>
</div>
```

and here is a custom @{whateverName} test

```html
<pre @hi> hello</pre>
```

Looks like somehow breeze is removing @click (and eny @{name} ) param from html.

I've confirmed that, becuase when I Disable Breeze for specified URL

<img width="580" alt="image" src="https://github.com/breezefront/community/assets/4760084/6bc0c3f2-6c62-4a55-aebe-31006cc85fd5">

@click  is rendered inside html element

<img width="688" alt="image" src="https://github.com/breezefront/community/assets/4760084/85ff1139-12fd-41f5-b9d9-a6697f5e8069">

could this be a conflict with any breeze js? or is module overriding template output?

thanks in advance

**Comment by JDavidVR** [comment]
update: looks like this class \Swissup\Breeze\Plugin\ViewResultPage is removing the @click from html elements

**Comment by JDavidVR** [comment]
confirmed, it is removed on this line code:

https://github.com/breezefront/module-breeze/blob/master/Model/Filter.php#L53

```
  foreach ($this->domFilters as $filter) {
            $filter->process($document);
        }
```

**Comment by vovayatsyuk** [comment, marked as answer]
Hi,

Looks like DomDocument removed this attribute. Can you please try if the following patch will help:

```diff
--- swissup/module-breeze/Model/Filter.php
+++ swissup/module-breeze/Model/Filter.php
@@ -82,6 +82,7 @@
             '—' => 'bz-mdash',
             '“' => 'bz-ldquo',
             '”' => 'bz-rdquo',
+            '@' => 'bz-at',
         ];
     }
 

```

**Comment by JDavidVR** [comment]
Thanks now it;s working
